libs/author/website/src/lib/author-list/author-list.tsx

```diff
@@ -0,0 +1,88 @@
+import {useAuthorListQuery, AuthorListQuery, FullAuthorFragment} from '@wepublish/website/api'
+import {styled, css} from '@mui/material'
+import {QueryResult} from '@apollo/client'
+import {useEffect} from 'react'
+import {BuilderAuthorListProps, useWebsiteBuilder} from '@wepublish/website/builder'
+
+export const AuthorTileWrapper = styled('a')`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+  text-decoration: none;
+  color: inherit;
+`
+
+export const AuthorTileImageWrapper = styled('div')`
+  width: 240px;
+  height: 240px;
+`
+
+export const AuthorTileContentWrapper = styled('div')`
+  margin-top: 20px;
+`
+
+export const AuthorListWrapper = styled('article')`
+  display: grid;
+  grid-template-columns: repeat(4, minmax(230px, 1fr));
+  gap: ${({theme}) => theme.spacing(4)};
+  justify-items: center;
+`
+
+const imageStyles = css`
+  border-radius: 50%;
+`
+
+export type AuthorListProps = Pick<BuilderAuthorListProps, 'authors'> & {
+  onQuery?: (
+    queryResult: Pick<QueryResult<AuthorListQuery>, 'data' | 'loading' | 'error' | 'refetch'>
+  ) => void
+  skip?: number
+  take?: number
+} & BuilderAuthorListProps
+
+type AuthorTileProps = {
+  className?: string
+  author: FullAuthorFragment
+}
+
+export function AuthorList({onQuery, className, skip, take}: AuthorListProps) {
+  const {
+    elements: {Image, Paragraph, H6}
+  } = useWebsiteBuilder()
+
+  const AuthorTile = ({className, author}: AuthorTileProps) => (
```

Comment:
Please don't mix containers with view components. 
Components inside a component is also an antipattern as the reference is different on **every render**